packages/php-wasm/web/src/lib/load-runtime.ts

```diff
@@ -58,8 +59,8 @@ export async function loadWebRuntime(
 		);
 	}
 
-	if (loaderOptions.withICU) {
-		emscriptenOptions = withICUData(emscriptenOptions);
+	if (loaderOptions.withICU || loaderOptions.withIntl) {
```

Comment:
Is it time to deprecate and clean up`withICU` since it does the same thing as `withIntl`?